plugins/woocommerce/tests/e2e-pw/tests/merchant/create-restricted-coupons.spec.js

```diff
@@ -97,377 +97,389 @@ const test = baseTest.extend( {
-				} );
-			}
-			// set exclude sale items
-			if ( couponType === 'excludeSaleItems' ) {
-				await test.step( 'set exclude sale items coupon', async () => {
-					await page
-						.getByRole( 'link', {
-							name: 'Usage restriction',
-						} )
-						.click();
-					await page.getByLabel( 'Exclude sale items' ).check();
-				} );
-			}
-			// set product categories
-			if ( couponType === 'productCategories' ) {
-				await test.step( 'set product categories coupon', async () => {
-					await page
-						.getByRole( 'link', {
-							name: 'Usage restriction',
-						} )
-						.click();
-					await page
-						.getByPlaceholder( 'Any category' )
-						.pressSequentially( 'Uncategorized' );
-					await page
-						.getByRole( 'option', { name: 'Uncategorized' } )
-						.click();
-				} );
-			}
-			// set exclude product categories
-			if ( couponType === 'excludeProductCategories' ) {
-				await test.step( 'set exclude product categories coupon', async () => {
-					await page
-						.getByRole( 'link', {
-							name: 'Usage restriction',
-						} )
-						.click();
-					await page
-						.getByPlaceholder( 'No categories' )
-						.pressSequentially( 'Uncategorized' );
-					await page
-						.getByRole( 'option', { name: 'Uncategorized' } )
-						.click();
-				} );
-			}
-			// set products
-			if ( couponType === 'products' ) {
-				await test.step( 'set products coupon', async () => {
-					await page
-						.getByRole( 'link', {
-							name: 'Usage restriction',
-						} )
-						.click();
-					await page
-						.getByPlaceholder( 'Search for a product…' )
-						.first()
-						.pressSequentially( product.name );
-					await page
-						.getByRole( 'option', { name: product.name } )
-						.click();
-				} );
-			}
-			// set exclude products
-			if ( couponType === 'excludeProducts' ) {
-				await test.step( 'set exclude products coupon', async () => {
-					await page
-						.getByRole( 'link', {
-							name: 'Usage restriction',
-						} )
-						.click();
-					await page
-						.getByPlaceholder( 'Search for a product…' )
-						.last()
-						.pressSequentially( product.name );
-					await page
-						.getByRole( 'option', { name: product.name } )
-						.click();
-				} );
-			}
-			// set allowed emails
-			if ( couponType === 'allowedEmails' ) {
-				await test.step( 'set allowed emails coupon', async () => {
-					await page
-						.getByRole( 'link', {
-							name: 'Usage restriction',
-						} )
-						.click();
-					await page
-						.getByPlaceholder( 'No restrictions' )
-						.fill( couponData[ couponType ].allowedEmails[ 0 ] );
-				} );
-			}
-			// set usage limit
-			if ( couponType === 'usageLimitPerCoupon' ) {
-				await test.step( 'set usage limit coupon', async () => {
-					await page
-						.getByRole( 'link', { name: 'Usage limits' } )
-						.click();
+test.describe(
+	'Restricted coupon management',
+	{ tag: [ '@services', '@skip-on-default-wpcom' ] },
```

Comment:
Deleting coupons returns 500 for all coupon tests on WPCOM. There was https://github.com/woocommerce/woocommerce/pull/47474 to fix that, but it's still happening for this env. For now, I'm skipping these tests.

Coupons are deleted after the API call, but tests are failing with the 500 error.